src/lib/transition_frontier/full_frontier/full_frontier.ml

```diff
@@ -607,21 +600,18 @@ let apply_diffs t diffs ~ignore_consensus_local_state =
         let mutant, new_root =
           apply_diff t diff ~ignore_consensus_local_state
         in
-        t.hash <- Frontier_hash.merge_diff t.hash (Diff.to_lite diff) mutant ;
         update_metrics_with_diff t diff ;
         let new_root =
           match new_root with
           | None ->
               prev_root
           | Some state_hash ->
-              Some {state_hash; frontier_hash= t.hash}
+              Some {state_hash}
         in
         (new_root, Diff.Full.With_mutant.E (diff, mutant) :: diffs_with_mutants)
     )
   in
-  [%log' trace t.logger]
-    "Reached state %s after applying diffs to full frontier"
-    (Frontier_hash.to_string t.hash) ;
+  [%log' trace t.logger] "after applying diffs to full frontier" ;
```

Comment:
You can just kill this log now as it's only noise without the frontier hash.

Reply:
Thought it might be nice just to have for profiling potentially